Assets/_Scripts/BattleController.cs

```diff
@@ -0,0 +1,217 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public enum BattlePhase {
+    PLAYER,
+    ENEMY
+
+};
+
+
+public class BattleController : MonoBehaviour
+{
+    public List<RectTransform> battleOptionsUI;
+    public RectTransform selectionCursor;
+    public Vector2 selectionCursorOffset = new Vector2(0, 0);
+
+    public float maxTimeBeforeAction = 15;
+    public float playerActionCounter {get; set;}
+
+    public RectTransform ActionMenu;
+    public RectTransform CommentaryMenu;
+    public Text CommentaryText;
+
+
+    List<Player> _players;
+    Player _heroPlayer;
+
+
+    bool startActions = false;
+
+
+    void Start() {
+        _players = GameManager.Instance.party.GetPlayersInPosition();
```

Comment:
Do players ever get updated when party changes?

Reply:
I was thinking about changing that later. Maybe instead I'll make a helper function that calls GameManager.Instance.party.GetPlayersInPosition() each time. It won't be as efficient, but the size of the list is 4 so it shouldn't matter.